current folder:
/home/nngu0068/vs25/nngu0068/FYP/fyp_2023
Collecting pandas
  Downloading pandas-1.3.5-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.3/11.3 MB 20.7 MB/s eta 0:00:00
Collecting pytz>=2017.3
  Downloading pytz-2023.2-py2.py3-none-any.whl (502 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.1/502.1 kB 953.6 kB/s eta 0:00:00
Collecting python-dateutil>=2.7.3
  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 3.8 MB/s eta 0:00:00
Requirement already satisfied: numpy>=1.17.3 in ./.venv/lib/python3.7/site-packages (from pandas) (1.21.6)
Requirement already satisfied: six>=1.5 in ./.venv/lib/python3.7/site-packages (from python-dateutil>=2.7.3->pandas) (1.16.0)
Installing collected packages: pytz, python-dateutil, pandas
Successfully installed pandas-1.3.5 python-dateutil-2.8.2 pytz-2023.2
Traceback (most recent call last):
  File "dna_train.py", line 11, in <module>
    from gen_data_func import *
  File "/monfs01/projects/vs25/nngu0068/FYP/fyp_2023/gen_data_func.py", line 18, in <module>
    import randomizeSequence as rSeq
  File "/monfs01/projects/vs25/nngu0068/FYP/fyp_2023/randomizeSequence.py", line 3, in <module>
    from matplotlib.pyplot import legend
ModuleNotFoundError: No module named 'matplotlib'
